Makefile

```diff
@@ -42,7 +42,7 @@ ifeq ($(DEBUG),1)
 endif
 
 ifneq ($(SANITIZER),none)
-	INTERNAL_OPTIONS += -Db_sanitize=$(SANITIZER) -Db_lundef=false
+	INTERNAL_OPTIONS += $(addprefix, -Db_sanitize=,$(SANITIZER)) -Db_lundef=false
```

Comment:
I have one question here: why `addprefix`? I notice below that you're specifying multiple sanitizers, which would translate to multiple option calls being used. But I'm guessing that only one will actually be picked up by Meson (i.e., multiple uses of the option would result in only the first or last being used)

Reply:
(I'm asking because maybe I'm missing something obvious!)

Reply:
Ah, I didn't read the meson docs :grimacing: I'll fix it.

Reply:
https://mesonbuild.com/Release-notes-for-0-42-0.html#simultaneous-use-of-address-and-undefined-behavior-sanitizers for future me to reference :sweat_smile: